### ISSUE TITLE
False warning when `draw_ids` argument is used with `brms` models

### ISSUE THREAD

**Issue by mattansb** [vincentarelbundock/marginaleffects#826]
The inclusion of the `draw_ids` argument triggers a warning - even though it is used properly.

```R
library(brms)
library(marginaleffects)

mod <- brm(mpg ~ hp, data = mtcars)

avg_slopes(mod, variables = "hp")
#> 
#>  Term Estimate   2.5 %  97.5 %
#>    hp  -0.0682 -0.0889 -0.0481
#> 
#> Columns: term, estimate, conf.low, conf.high 

avg_slopes(mod, variables = "hp", draw_ids = 1:100)
#> 
#>  Term Estimate   2.5 %  97.5 %
#>    hp  -0.0678 -0.0893 -0.0445
#> 
#> Columns: term, estimate, conf.low, conf.high 
#> 
#> Warning messages:
#> 1: These arguments are not supported for models of class `brmsfit`: draw_ids. Valid arguments include: ndraws, re_formula, allow_new_levels, sample_new_levels, dpar, resp. Please file a request on Github if you believe that additional arguments should be supported: https://github.com/vincentarelbundock/marginaleffects/issues 
#> 2: These arguments are not supported for models of class `brmsfit`: draw_ids. Valid arguments include: ndraws, re_formula, allow_new_levels, sample_new_levels, dpar, resp. Please file a request on Github if you believe that additional arguments should be supported: https://github.com/vincentarelbundock/marginaleffects/issues
```

```R
sessionInfo()
#> R version 4.2.2 (2022-10-31 ucrt)
#> Platform: x86_64-w64-mingw32/x64 (64-bit)
#> Running under: Windows 10 x64 (build 22621)
#> 
#> Matrix products: default
#> 
#> locale:
#> [1] LC_COLLATE=English_Israel.utf8  LC_CTYPE=English_Israel.utf8    LC_MONETARY=English_Israel.utf8 LC_NUMERIC=C                    LC_TIME=English_Israel.utf8    
#> 
#> attached base packages:
#> [1] stats     graphics  grDevices utils     datasets  methods   base     
#> 
#> other attached packages:
#> [1] marginaleffects_0.13.0 brms_2.19.0            Rcpp_1.0.10           
#> 
#> loaded via a namespace (and not attached):
#>   [1] TH.data_1.1-2        colorspace_2.1-0     ellipsis_0.3.2       rprojroot_2.0.3      estimability_1.4.1   cranlike_1.0.2       markdown_1.7         base64enc_0.1-3     
#>   [9] rstudioapi_0.14      farver_2.1.1         rstan_2.26.13        remotes_2.4.2        DT_0.28              bit64_4.0.5          fansi_1.0.4          mvtnorm_1.2-2       
#>  [17] bridgesampling_1.1-2 codetools_0.2-18     splines_4.2.2        cachem_1.0.8         knitr_1.43           shinythemes_1.2.0    bayesplot_1.10.0     jsonlite_1.8.5      
#>  [25] shiny_1.7.4          compiler_4.2.2       emmeans_1.8.6        backports_1.4.1      Matrix_1.5-4.1       fastmap_1.1.1        cli_3.6.1            later_1.3.1         
#>  [33] htmltools_0.5.5      prettyunits_1.1.1    tools_4.2.2          igraph_1.4.3         coda_0.19-4          gtable_0.3.3         glue_1.6.2           reshape2_1.4.4      
#>  [41] dplyr_1.1.2          posterior_1.4.1.9000 rappdirs_0.3.3       V8_4.3.0             vctrs_0.6.2          debugme_1.1.0        nlme_3.1-160         crosstalk_1.2.0     
#>  [49] insight_0.19.2       tensorA_0.36.2       xfun_0.39            stringr_1.5.0        ps_1.7.5             collapse_1.9.6       mime_0.12            parsedate_1.3.1     
#>  [57] miniUI_0.1.1.1       lifecycle_1.0.3      gtools_3.9.4         MASS_7.3-58.1        zoo_1.8-12           scales_1.2.1         colourpicker_1.2.0   promises_1.2.0.1    
#>  [65] Brobdingnag_1.2-9    parallel_4.2.2       sandwich_3.0-2       inline_0.3.19        rematch2_2.1.2       shinystan_2.6.0      curl_5.0.1           memoise_2.0.1       
#>  [73] gridExtra_2.3        ggplot2_3.4.2        loo_2.6.0            StanHeaders_2.26.26  stringi_1.7.12       RSQLite_2.3.1        dygraphs_1.1.1.6     desc_1.4.2          
#>  [81] checkmate_2.2.0      crancache_0.0.0.9001 pkgbuild_1.4.0       cmdstanr_0.5.3       rlang_1.1.1          pkgconfig_2.0.3      matrixStats_1.0.0    distributional_0.3.2
#>  [89] lattice_0.20-45      rstantools_2.3.1     htmlwidgets_1.6.2    bit_4.0.5            processx_3.8.1       tidyselect_1.2.0     plyr_1.8.8           magrittr_2.0.3      
#>  [97] R6_2.5.1             generics_0.1.3       multcomp_1.4-24      DBI_1.1.3            pillar_1.9.0         withr_2.5.0          xts_0.13.1           survival_3.4-0      
#> [105] abind_1.4-5          tibble_3.2.1         crayon_1.5.2         utf8_1.2.3           grid_4.2.2           data.table_1.14.8    blob_1.2.4           callr_3.7.3         
#> [113] threejs_0.3.3        digest_0.6.31        xtable_1.8-4         httpuv_1.6.11        RcppParallel_5.1.7   stats4_4.2.2         munsell_0.5.0        shinyjs_2.1.0
```

**Comment by vincentarelbundock** [comment]
Thanks!

Should be added to the white list here: https://github.com/vincentarelbundock/marginaleffects/blob/main/R/sanity_dots.R#L30

**Comment by vincentarelbundock** [comment]
Thanks, should be fixed in 0.13.0.9001 on Github
